scripts/build

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+# execute all of the steps necessary to build the project
+set -e #-x
+
+usage() {
+  echo "usage: $0 [-h|--help] COMMAND"
+  echo ""
+  echo "[-h|--help]      print this help text"
+  echo ""
+  echo "COMMAND is one of the following:"
+  echo "- test           execute the tests (should be relatively fast)"
+  echo "- integration    execute the integration tests"
+  echo "- project        the default build command"
+  echo "- docker         build a docker image"
+  echo "- checkFormat    check formatting of files"
+  echo "- fixFormat      automatically fix formatting of files"
+}
+
+# script variables and defaults
+source $(dirname $0)/init.sh
+
+DEBUG_OPTS=""
```

Comment:
I don't think this is used anywhere - feel free to remove. 
intention is/was to add command-line options to be able to hook up a debugger to a remotely running instance (but never got there)